package.json

```diff
@@ -36,7 +38,7 @@
     "url": "git@github.com:jitsi/spot-electron-sdk.git"
   },
   "scripts": {
-    "build": "rm -rf lib/* && tsc",
+    "build": "tsc",
```

Comment:
what prompted this change?

Reply:
Building the package on windows

Reply:
Maybe we should use rimraf? https://github.com/isaacs/rimraf